source/_integrations/unifiprotect.markdown

```diff
@@ -295,6 +344,38 @@ The UniFi Protect integration provides support for various event types triggered
   - **nfc_id**: The ID of the scanned NFC card.
 - **Description**: This event is triggered when an NFC card is scanned at a compatible device (e.g., a smart doorbell). It contains information such as the `nfc_id` of the scanned card.
 
+#### Example G4 Doorbell NFC Scanned Automation
+
+```yaml
+alias: G4 Doorbell NFC Scanned Automation
+description: >-
+  Automation that triggers when a specific NFC card is scanned on the G4
+  Doorbell Pro
+triggers:
+  - event_type: state_changed
+    event_data:
+      entity_id: event.g4_doorbell_pro_poe_nfc  # Replace with your doorbell entity
+    trigger: event
+conditions:
+  - condition: template
+    value_template: >
+      {% raw %}{{ 
+         trigger.event.data.new_state is not none and
+         trigger.event.data.new_state.attributes.event_type == 'scanned' and
+         trigger.event.data.new_state.attributes.nfc_id in ['ABCDEF1234', 'OTHER_ALLOWED_ID']
+       }}{% endraw %}
+actions:
+  - data:
+      message: >-
+        {% raw %}The NFC card with ID {{ trigger.event.data.new_state.attributes.nfc_id }} has been scanned at the doorbell.{% endraw %}
+      title: NFC Scan Notification
+    action: notify.mobile_app_your_device # Replace with your notification target
+```
+
+**Warning:**
+
+When processing NFC scans, always validate the scanned ID. The scan event is also triggered by unknown NFC cards. Additionally, this event was developed using third-party cards, as the developer did not have access to official UniFi cards at the time. With third-party cards, the scan relies on the card's serial number. While this approach is not uncommon, it is important to note that the card's serial number is generally not considered a secure identifier and can be duplicated relatively easily.
```

Comment:
```suggestion
When processing NFC scans, always validate the scanned ID. Unknown NFC cards also trigger the scan event. Additionally, this event was developed using third-party cards, as the developer did not have access to official UniFi cards at the time. With third-party cards, the scan relies on the card's serial number. While this approach is not uncommon, it is essential to note that the card's serial number is generally not considered a secure identifier and can be duplicated relatively easily.
```

I also tested it with third party cards (used a Telsa key card) 🤣

Reply:
@bdraco Haha, I tested it with "Mifare Classic 1k" Clones, specifically cards designed for this purpose that allow you to overwrite the normally non-writable sector 0, including the serial number, to clone cards. :D I still had some of those lying around here, so I quickly checked with the Flipper to see if the card's UUID is what comes through the API—and that's exactly what it was.